content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -177,46 +209,102 @@ spec:
 
 ```
 
-The above template defines parameters for each job object using a list of
-python dicts (lines 1-4).  Then a for loop emits one job yaml object
-for each set of parameters (remaining lines).
-We take advantage of the fact that multiple yaml documents can be concatenated
-with the `---` separator (second to last line).
-.)  We can pipe the output directly to kubectl to
-create the objects.
+The above template defines two parameters for each Job object using a list of
+python dicts (lines 1-4).  Then a for loop emits one Job manifest for each
```

Comment:
nit: line 213, extra space, `.  Then`. Could remove `Then`. A for loop emits ...